apiary.apib

```diff
@@ -4964,47 +4964,54 @@ Dashboard data can be retrieved using:
                 }
             ],
             "reviewListings": [
+                
                 {
-                    "name": "revlocal",
-                    "url": "http://birdeye.com/maria-da-vittorio-943999808"
-                },
-                {
+		    "id": 3,
                     "name": "Citysearch",
                     "url": "http://www.citysearch.com/biz/maria-da-vittorio-san-francisco"
                 },
                 {
+		    "id": 2,
```

Comment:
@sachindar can you just move this as first entry in the array? just want the "id" to be arranged serially.